### PR TITLE
Upgrade pytest-aiohttp

### PR DESCRIPTION
**Pull request by DoctorJohn**
## Description

#1615 needed some extra work (I did the upgrade on one of my packages the other day).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Closes #1615 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: strawberry/aiohttp/handlers/http_handler.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ async def handle(self) -> web.StreamResponse:
     async def get(self, request: web.Request) -> web.StreamResponse:
         if self.should_render_graphiql(request):
             return self.render_graphiql()
-        return web.HTTPNotFound()
+        raise web.HTTPNotFound()
```
Comment (DoctorJohn) [inline strawberry/aiohttp/handlers/http_handler.py:40]: Returning http exceptions has been deprecated.

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/), so make sure to pick the appropriate type. If in doubt feel free to ask :)


Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Jonathan Ehwald for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

File: strawberry/aiohttp/handlers/http_handler.py (hunk 1 above)
Comment (patrick91) [reply strawberry/aiohttp/handlers/http_handler.py:40]: should we make a patch release for this?

Comment (DoctorJohn) [reply strawberry/aiohttp/handlers/http_handler.py:40]: i don't think thats necessary. this change doesn't really affect users of the library.

Comment (patrick91) [reply strawberry/aiohttp/handlers/http_handler.py:40]: mostly because we are going to do a release before the deprecated thing gets removed? :)

**Pull request merged**